src/Pages/Home/home.js

```diff
@@ -32,23 +33,72 @@ const Home = () => {
           marginTop: "5rem",
         }}
       >
+        <div className="container-hero">
+          <img
+            className="image-hero"
+            src="https://cdn.pixabay.com/photo/2021/06/03/01/38/spices-6305695__480.jpg"
+            loading="lazy"
+            alt="hero-image"
+          />
+          <div
+            className="text-hero"
+            style={{
+              display: "flex",
+              flexDirection: "column",
+              justifyContent: "center",
+              alignItems: "center",
+              gap: "1rem",
+            }}
+          >
+            <p className="font-bold text-3xl">
+              Learn Best Food Receipes with top
+            </p>
+            <p className="font-bold text-3xl">videos from WOW TUBE</p>
+            <Link className="btn-hero font-bold text-base" to="/videos">
+              Videos
+            </Link>
+          </div>
+        </div>
+        <div className="spacer-3"></div>
+        <h2 style={{ color: "white", fontWeight: "500" }} className="spacer-4">
```

Comment:
You Can declare Similar styling in one place!

Reply:
Okay. Will do that. Thankyou for reviewing